students/skowronski_rafal/lesson_08_regular_expressions/regex.py

```diff
@@ -0,0 +1,85 @@
+import re
+import copy
+import enum
+
+
+class PostalCodeKind(enum.Enum):
+    POLISH = 1,
+    AMERICAN = 2
+
+
+def strip(text: str, chars: str=None) -> str:
+    if chars is None:
+        chars = r'\s'
+
+    regex = re.compile('^([{0}])*(.*?)([{0}])*$'.format(chars))
+    match = regex.search(text)
+    if match is None:
+        return copy.copy(text)
+
+    return match.group(2)
+
+
+def is_password_strong(text: str) -> bool:
+    regex = re.compile(r'^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.{8,})')
```

Comment:
This regexp is long, we can move it to separate value like:
PASSWORD_REGEXP -> later we can use it once again :)